<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trailClass</class>
 <widget class="QMainWindow" name="trailClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1622</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sort-visualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="cellsToSort">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>700</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="comparisonsLabel">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="algorithmList">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Bubble Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heap Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quick Sort</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="sortButton">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>sort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1622</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="trail.qrc"/>
 </resources>
 <connections/>
</ui>
